test.js

```diff
@@ -49,3 +49,22 @@ it('should generate source maps', function (cb) {
 
 	init.end();
 });
+
+it('should list used helpers in file.babel', function (cb) {
+	var stream = to5();
+
+	stream.write(new gutil.File({
+		cwd: __dirname,
+		base: __dirname + '/fixture',
+		path: __dirname + '/fixture/fixture.es6',
+		contents: new Buffer('class MyClass {};')
+	}));
+
+	stream.on('data', function (file) {
```

Comment:
Can this event handler be moved before `stream.write` to conform with the style of the other unit tests.